src/FileDropzone/FileDropzone.js

```diff
@@ -23,110 +20,92 @@ import {
 export type Props = {
   value: ?File,
   onFileChange: (File, setSubmitting: SetSubmitting) => void | Promise<void>,
-  placeholder?: string,
-  browse?: string,
-  dot?: string,
-  accept?: string,
-  height?: number,
-};
-type InnerProps = {
-  value: ?File,
-  onFileChange: (File, setSubmitting: SetSubmitting) => void | Promise<void>,
-  placeholder?: string,
-  browse?: string,
-  dot?: string,
-  accept?: string,
+  placeholder: string,
+  browse: string,
+  dot: string,
+  accept: string,
```

Comment:
Are these 4 props `placeholder`, `browse`, `dot`, `accept` still optional?
Or flow-bin has new behavior? 

Thought this would result _⚠️ flow check error_ in [`it's API story`](https://github.com/Mediatek-Cloud/mcs-ui/blob/master/src/FileDropzone/FileDropzone.example.js#L37)

Reply:
Flow handle that for you. It is a normal use case in [Using Default Props for Functional Components](https://flow.org/en/docs/react/components/#toc-using-default-props-for-functional-components) section which is not a new behavior.